drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -241,6 +251,15 @@ static int _atwinc15x0_send(netdev_t *netdev, const iolist_t *iolist)
         return -ENODEV;
     }
 
+    /* send wakes from standby but not from sleep */
```

Comment:
Why not to wake up the WiFi when the application wants to send something?

Reply:
tbh I didn't know what to make out of the distinction between the STANDBY and the SLEEP state, so this was my interpretation of what could make sense.

I can drop the STANDBY state of course to simplify things.

Reply:
Hmm, good question. What would an application expect after putting a `netdev` into sleep mode explicitly? Should it explicitly wake it up before anything can be sent or should the device wake up as soon as there is something to send? I am unsure about that as well.

Reply:
Not waking up automatically makes sense as otherwise there would be a lot of spurious wakes by e.g. router advertisements.